Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
 	"bee-api/bee-rest-api",
+	"bee-autopeering",
```

Comment:
Can we maybe nest that in bee-network ? So have
`bee-network/bee-network` and
`bee-network/bee-autopeering` ?

Reply:
I am not 100% sure it makes as much sense in the chrysalis node that it does in the coordicide node but if you want, sure